source/codegen/metadata/nidmm/config.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# This file is generated from NI-DMM API metadata version 19.6.0d7
+config = {
+    'api_version': '19.6.0d7',
+    'c_header': 'nidmm.h',
+    'c_function_prefix': 'niDMM_',
+    'service_class_prefix': 'NiDMM',
+    'java_package': 'com.ni.grpc.dmm',
+    'csharp_namespace': 'NationalInstruments.Grpc.DMM',
+    'namespace_component': 'nidmm',
+    'close_function': 'close',
```

Comment:
You will have to change this to 'Close' after adding the 'cname' tag for it

Reply:
changed that